src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftExcludeTranspilation.md

```diff
@@ -0,0 +1,264 @@
+This document goes over one of the more complicated rewrites for PartiQL's `EXCLUDE` to Redshift.
```

Comment:
self-review: would recommend starting with this doc to better understand the end-to-end rewrite